packages/suite/src/components/wallet/TransactionItem/TransactionHeading.tsx

```diff
@@ -168,12 +163,12 @@ export const TransactionHeading = ({
                         />
                     )}
                     <BlurWrapper $isBlurred={isPhishingTransaction}>
-                        <HeaderWrapper>
+                        <Row gap={spacings.xxs}>
```

Comment:
Please use `Box` here. Row is considered for multiple children, and Box is universal building block for cases like this one.

Reply:
In this case there can be more children depending on a condition, but I'll note that for other cases